### PR TITLE
⚗️ Start new exp for /civic with referrer logo

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/pages/civic/index.vue [hunk 1]
```diff
@@ -147,8 +142,12 @@
                 .flex.justify-center.mb-24(
                   v-if="isExperimenting"
                 )
-                  LikeButtonAnimation(:size="180")
+                  LikeButtonAnimation(
+                    :size="180"
+                    :avatar="referrer.avatar"
```
Comment (williamchong007) [inline src/pages/civic/index.vue]: will this explode if referrer is undefined?

File: src/pages/civic/index.vue [hunk 2]
```diff
@@ -147,8 +142,12 @@
...
+                  )
                   LcChopCivicLiker(
+                    v-if="!referrer.avatar"
```
Comment (williamchong007) [inline src/pages/civic/index.vue]: will this explode if referrer is undefined?

File: src/pages/civic/index.vue [hunk 3]
```diff
@@ -237,7 +236,12 @@ export default {
   },
   mixins: [
     IntercomMixinFactory({ isBootAtMounted: false }),
-    experimentMixin('isExperimenting', 'civic-page', 'variant'),
+    experimentMixin(
+      'isExperimenting',
+      'civic-page',
+      'variant',
+      that => !!that.referrer
```
Comment (nwingt) [inline src/pages/civic/index.vue]: I have guarded in L243

File: src/pages/civic/index.vue (hunk 1 above)
Comment (nwingt) [reply src/pages/civic/index.vue]: I have guarded in L243

File: src/pages/civic/index.vue (hunk 2 above)
Comment (nwingt) [reply src/pages/civic/index.vue]: I have guarded in L143 & L243

File: src/pages/civic/index.vue (hunk 1 above)
Comment (williamchong007) [reply src/pages/civic/index.vue]: After we end the experiment, L243 would be removed, and this part of template might be applied.
Should also guard in template?

**Pull request merged**